indicators/roblox-phishing.yml

```diff
@@ -0,0 +1,35 @@
+title: Roblox phishing

```

Comment:
Please follow the rule naming convention specified in `CONTRIBUTING.md`. Typically the rule title and file name are the same.